producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/submitquery/SubmitQueryConfigFive.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.moj.sdt.producers.comx.config.submitquery;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.moj.sdt.domain.ErrorLog;
+import uk.gov.moj.sdt.domain.SubmitQueryRequest;
+
+@Configuration
+@EnableAutoConfiguration
+public class SubmitQueryConfigFive {
```

Comment:
Perhaps change name to SubmitQueryConfigMcolDefenceFive just to make it more descriptive of the purpose (and in case any alternative query configs are introduced in the future).  If this change is made then apply the same change to the other four query config classes.

Reply:
done